engine/transcoder/ffmpeg.go

```diff
@@ -16,6 +16,11 @@ type Transcoder interface {
 }
 
 func New() Transcoder {
+	path, err := exec.LookPath("ffmpeg")
+	if err != nil {
+		log.Error("Unable to find ffmpeg: scanning and transcoding won't work!")
+	}
+	log.Debug("Found ffmpeg: ", path)
```

Comment:
Navidrome's log package needs a key/value pair for extra parameters, to keep consistency. It needs to be something like this:
```
log.Debug("Found ffmpeg", "path", path)
```

The only exception is when the last param is an `error` instance, as above